gluoncv/auto/estimators/yolo.py

```diff
@@ -0,0 +1,425 @@
+"""YOLO Estimator"""
+import os
+import time
+import warnings
+from collections import OrderedDict
+import numpy as np
+
+import logging
+import mxnet as mx
+from mxnet import gluon
+from mxnet import autograd
+from gluoncv import data as gdata
+from ... import utils as gutils
+from ...model_zoo import get_model
+from ...data.batchify import Tuple, Stack, Pad
+from ...utils.metrics.accuracy import Accuracy
+from ...utils import LRScheduler, LRSequential
+from ...model_zoo.center_net import get_center_net, get_base_network
+from ...loss import MaskedL1Loss, HeatmapFocalLoss
+
+from ...data.transforms.presets.yolo import YOLO3DefaultTrainTransform
+from ...data.transforms.presets.yolo import YOLO3DefaultValTransform
+from ...data.dataloader import RandomTransformDataLoader
+from ...utils.metrics.voc_detection import VOC07MApMetric
+from ...utils.metrics.coco_detection import COCODetectionMetric
+
+from ..data.coco_detection import coco_detection, load_coco_detection
+from .base_estimator import BaseEstimator, set_default
+
+from mxnet.contrib import amp
+try:
+    import horovod.mxnet as hvd
+except ImportError:
+    hvd = None
+
+from sacred import Experiment, Ingredient
+
+__all__ = ['YoloEstimator']
+
+yolo_net = Ingredient('yolo_net')
```

Comment:
maybe `yolo3` is better? `yolo_net` is ambiguous

Reply:
updated. thanks